cunumeric/random/random.py

```diff
@@ -1086,6 +1086,20 @@ def randint(
     --------
     Multiple GPUs, Multiple CPUs
     """
+
+    if not isinstance(low, int):
+        raise NotImplementedError("'low' must be an integer")
+    if high is not None and not isinstance(high, int):
+        raise NotImplementedError("'high' must be an integer or None")
+
+    if high is None:
+        low = 0
+        high = low
+    if low >= high:
+        raise ValueError("low >= high")
```

Comment:
Seems like if `high is None` then this error is guaranteed to happen since then `high = low = 0`

Reply:
Yes, rookie mistake on my part, [e0cca81](https://github.com/nv-legate/cunumeric/pull/966/commits/e0cca81d68ec2e1c92be65171deb3903d7ec21dd)